### PR TITLE
Expose `type` in `ContextEntry`

### PR DESCRIPTION
**Pull request by SimonDegraeve**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -18,13 +18,17 @@ export type TypeOf<RT> = ExtractType<*, RT>;
...
 
-export type ContextEntry = {
+export type ContextEntry<T> = {
   key: string,
-  name: string
+  type: Type<T>,
 };
 
-export type Context = Array<ContextEntry>;
+export type Context = Array<ContextEntry<any>>;
```
Comment (SimonDegraeve) [inline src/index.js:31]: Not sure about `any` here.

Comment (gcanti) [reply src/index.js:31]: `*` doesn't work? `mixed`?

Comment (SimonDegraeve) [reply src/index.js:31]: both not working

Comment (gcanti) [reply src/index.js:31]: Let's go for `any` then. We can't show anymore the "No `any` type here" badge in the readme :)

**Comment by gcanti** [comment]
Thanks!

**Pull request merged**